src/containers/layouts/articles/article3/article3.component.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import {
+  ImageBackground,
+  ScrollView,
+  View,
+  ViewProps,
+} from 'react-native';
+import {
+  ThemedComponentProps,
+  ThemeType,
+  withStyles,
+} from '@kitten/theme';
+import {
+  Text,
+  Comment,
+  CommentProps,
+} from '@src/components/common';
+import { Input } from '@kitten/ui';
+import { Article } from '@src/core/model';
+
+interface ComponentProps {
+  article: Article;
+  comments: string[];
+  comment: string;
+  onCommentSubmit: () => void;
+  onCommentTextChange: (text: string) => void;
+  onCommentPress: () => void;
+  onLikePress: () => void;
+}
+
+export type Article3Props = ThemedComponentProps & ComponentProps;
+
+class Article3Component extends React.Component<Article3Props> {
+
+  private onCommentButtonPress = () => {
+    this.props.onCommentPress();
+  };
+
+  private onLikeButtonPress = () => {
+    this.props.onLikePress();
+  };
+
+  private onCommentTextChange = (text: string): void => {
+    this.props.onCommentTextChange(text);
+  };
+
+  private handleTextSubmit = () => {
+    this.props.onCommentSubmit();
+  };
+
+  private renderComment = (comment: string, key: number): React.ReactElement<CommentProps> => {
+    const { article } = this.props;
+
+    return (
+      <Comment
+        key={key}
+        author={article.author}
+        comment={comment}
+        date={article.date}
+        commentsCount={45}
+        likes={22}
+        onLikePress={() => {
```

Comment:
prefer creating empty handling functions inside component

Reply:
Will be refactored in the next PR.